test/support/cog_test_utils.ex

```diff
@@ -38,19 +39,16 @@ defmodule Mobius.CogTestUtils do
     end)
   end
 
-  @doc """
-  Asserts that a message was sent with the given expected body
-
-  The ordering of properties in the expected body matters.
-  `%{content: "Content", tts: false}` won't match a message sent with
-  `%{tts: false, content: "Content"}`
-  """
-  @spec assert_message_sent(Message.message_body()) :: :ok
-  def assert_message_sent(message) do
-    json_message = Jason.encode!(message)
-    %Tesla.Env{url: url} = assert_receive %Tesla.Env{body: ^json_message, method: :post}
-    # This only works if the channel_id used to send the message was `nil`
-    assert String.ends_with?(url, "/channels/:channel_id/messages")
-    :ok
+  @doc "Asserts that a message was sent returns the body of the request"
```

Comment:
```suggestion
  @doc "Asserts that a message was sent and returns the body of the request"
```
The end isn't quite true, the body isn't returned, but merely matched against the expectation. I think that just saying "and matches the body of the request against the expectation" might not be clear enough even though it's technically more accurate. What do you think?

Reply:
Yeah I changed what this function does like 4 times while doing this PR and probably forgot to update the doc at some point 😅

It does however return the body of the request because the `assert unquote(x) = body` is the last thing in the quote which means `body = assert_message_sent(%{})` does in fact have the body of the request in the variable `body`. It's not really the expected way to use this macro though, I agree.

I think something like "Asserts and matches on the body of a message send request" with an example might be good enough? I don't really see how to be clear and concise here so I'd embrace it by providing examples instead.

Reply:
You're right, I forgot `assert` actually returns the result! And I agree, examples are probably the way to go here.